client/my-sites/media-library/filter-bar.jsx

```diff
@@ -38,7 +38,8 @@ export class MediaLibraryFilterBar extends Component {
 		basePath: '/media',
 		onFilterChange: noop,
 		onSearch: noop,
-		translate: identity
+		translate: identity,
+		source: '',
```

Comment:
The `source` and `onSourceChange` props haven't been defined in the propTypes array. I understand that, in an upcoming PR, you plan to create a new kind of list component within FilterBar to take care of selecting the different sources, and that component will fire the `onSourceChange`.